helper-cli/src/main/kotlin/commands/packagecuration/SplitCommand.kt

```diff
@@ -64,14 +64,13 @@ internal class SplitCommand : CliktCommand(
 
         val mapper = createBlockYamlMapper()
         curationsByOutputFile.forEach { (outputFile, curations) ->
-            val curationsToPersist = readPackageCurations(outputFile).toMutableSet()
-
-            curationsToPersist += curations
-
-            val curationsWithBlockComment = curationsToPersist.mapTo(mutableSetOf()) { it.formatComment() }
+            val curationsToPersist = (readPackageCurations(outputFile) + curations)
+                .map { it.formatComment() }
+                .distinct()
```

Comment:
Why is `.distinct()` better than just using `.toSet()`?
Especially because `.distinct()` uses `toSet()` internally and then converts it to a `List` again.

Reply:
Due to the sorting right below it becomes a list anyway.
I found distinct slightly more speaking and a bit more emphasizing that this deals with a list, not a set.

Do you want me to change it?

Reply:
> I found distinct slightly more speaking

I agree.

> Do you want me to change it?

No.